plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/block.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "woocommerce/add-to-cart-with-options",
+	"version": "1.0.0",
+	"title": "Add to Cart with Options (Blockified)",
+	"description": "Display a button so the customer can add a product to their cart. Options will also be displayed depending on product type. e.g. quantity, variation.",
+	"category": "woocommerce-product-elements",
+	"attributes": {
+		"isDescendentOfSingleProductBlock": {
```

Comment:
I think we should try to find a better way to detect when the block is a descendent of a Single Product block.
It pollutes the block.

Reply:
I created the issue https://github.com/woocommerce/woocommerce/issues/53284 to tackle this problem.